OpenDreamRuntime/Procs/DMOpcodeHandlers.cs

```diff
@@ -1477,6 +1477,32 @@ static class DMOpcodeHandlers {
             return null;
         }
 
+        ///<summary>Right now this is used exclusively by addtext() calls, to concatenate its arguments together,
+        ///but later it might make sense to have this be a simplification path for detected repetitive additions of strings,
+        ///so as to slightly reduce the amount of re-allocation taking place.
+        ///</summary>.
+        public static ProcStatus? MassConcatenation(DMProcState state)
+        {
+            int count = state.ReadInt();
+            if (count == 1) // One argument -- shouldn't really ever happen. addtext() compiletimes with one arg and stringification should probably be a different opcode
+            {
+                Logger.Warning("addtext() called with one argument at runtime."); // TODO: tweak this warning if this ever gets used for other sorts of string concat
+                state.Push(new DreamValue());
```

Comment:
`new DreamValue()` leaves all the struct's values uninitialized, which is an invalid DreamValue.
```suggestion
                state.Push(DreamValue.Null);
```

Reply:
Y'know I *did* actually see that Null static when I was scanning through the OOP, but it didn't click like "oh, wait, *that's* how you initialize to null" 😅